### PR TITLE
updates layout and missing thumbs

### PR DESCRIPTION
**Pull request by straleyb**

### CODE REVIEW

**Comment by straleyb** [comment]
There are still questions that need to be answered such as:
"Are we still doing the first time here and dig deeper links" and
"Do we want the masthead image to be changing on rotation or is it fine just staying that one image (blimp image)"

**Comment by jsimic** [comment]
@straleyb Yes, we still want the First time here? and Dig deeper sections. Yes, we want a rotating masthead image.

File: app/views/hyrax/homepage/_home_content.html.erb [hunk 1]
```diff
@@ -11,14 +11,73 @@
         <% @member_count = @member_docs.length %>
         <% @collection_solr_doc = SolrDocument.find(collection.id) %>
         <span class="featured-collection-row flex-container">
-        <% @member_works[0..2].each do |work| %>
+        <% if @member_works.length == 0 %>
```
[the remark this answers is not in the document]
Comment (CGillen) [reply app/views/hyrax/homepage/_home_content.html.erb:14]: This _**should**_ create an array like how we would want.
Take our array, pad it to 3 vals (or noop if >3 length already), grab the first 3, reverse it so nils are first, then choose placeholder or thumbnail depending on nil status.

The caveat here is that `#fill` mutates the original array. So you might need to call `#dup` on `@member_works` and work with a new array

**Review by CGillen — CHANGES_REQUESTED** [review]

File: app/views/hyrax/homepage/_home_content.html.erb (hunk 1 above)
[the remark this answers is not in the document]
Comment (straleyb) [reply app/views/hyrax/homepage/_home_content.html.erb:14]: Cool yeah ill test it out. Im not even sure how often we'll run into any featured collections with one work or two works in it so the default will run 99% of the time, but to get to wireframe consistent we needed to add the placeholders.

**Review by CGillen — APPROVED** [review]

**Pull request merged**